libs/blocks/caas/caas.css

```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
   align-self: baseline;
   margin-top: 6px;
 }
+
+
+/* collapsible panel */
+button.collapse-panel {
+  border: none;
+  font-size: 18px;
+  width: 26px;
+  height: 32px;
+  cursor: pointer;
+}
+
+button.collapse-panel:hover {
+background:#666;
+color: #eee;
+}
+
+.tool-content {
+  transition: all 400ms;
+  grid-template-columns: 340px 20px 1fr;
+  gap: 6px;
+}
+
+.panel-collapsed .config-panel {
+  z-index: -1;
```

Comment:
`z-index` works only for elements with a position other than `static` or if another stacking context property is used, which I don't see applied in this file, is is added in some other place?

Reply:
The element does not have static position. 
z-index in this context is preventing the mouse events (mouse over and clicks) to fire.

Reply:
Then why not use `pointer-events: none` to make the intention clear?